### ISSUE TITLE
Meaning of data in r9-models folder

### ISSUE THREAD

**Issue by wxuzhang** [jts/nanopolish#520]
For file etc/r9-models/r9.4_450bps.nucleotide.6mer.template.model, could you please help me explain meaning of level_mean , level_stdv, sd_mean, sd_stdv and weight for each 6 mer? Thanks in advance!

**Comment by jts** [comment]
Hi,

These values make up the "pore model" that describe the characteristics of the events we should observe for each k-mer. I describe this a little more in this blog post:

http://simpsonlab.github.io/2015/04/08/eventalign/

Jared

**Comment by wxuzhang** [comment]
Thanks for your reply! The post is clear for me on level_mean and level_stdv in the file. But I am still a little confuse about sd_mean and sd_stdv here.

This is my understanding about the data there:

For the data shown here:
#original_file	r9.4_180mv_450bps_6mer/template_median68pA.model
#kmer	level_mean	level_stdv	sd_mean	sd_stdv	weight
AAAAAA	86.486336	1.517846	0.941478	0.609357	4739.559092

level_mean: average of all AAAAAA signal event
level_stdv: standard deviation of all AAAAAA signal event

In each measurement, there are several data points. There will be a standard deviation for each event in that short duration.
sd_mean: means the average of those standard deviation
sd_stdv: means the standard deviation of those standard deviation

weight: means time accumulation in that state.

I am wondering whether I understand each parameter accurate. Thank you very much!

**Comment by jts** [comment]
Yes I think you are correct. A bit of background: the original HMM basecaller from ONT modeled the "spread" of the signal around the mean using an inverse gaussian distribution and `sd_mean` and `sd_stdv` are the parameters to this distribution. As far as I know most programs no longer model the signal spread (it was more useful when the reads were slower) so these fields are ignored. I don't know the precise definition of `weight` - it is something the training algorithm outputs - and I ignore it.
